microservices/google-api/google_search.py

```diff
@@ -382,30 +382,31 @@ def daterange(startDate, endDate):
 DROP TABLE IF EXISTS cmslite.google_pdt_old;
 
 CREATE TABLE IF NOT EXISTS cmslite.google_pdt_scratch (
```

Comment:
@mark-walle : Given that this is all done in a SQL transaction, do we need separate "pdt", "scratch", and "old" tables?

Reply:
Yes I believe you are correct since if any step of the execution plan failing, the transaction would just rollback.

I'll add a commit to simplify the transaction by removing the _old and _scratch tables; and just do everything on the `google_pdt` table directly.

Reply:
@mark-walle : Sounds good.

Reply:
Looks like this was done in a commit below. Resolving the thread.